### PR TITLE
Update Teleport doc with latest instructions

### LINKED ISSUE #2939: Add instructions for Kusama/Statemine Teleports
Given the recent changes to Polkadot JS Apps interface and the questions pertaining to Teleports, update the Wiki with the latest instructions

### PR DESCRIPTION
**Pull request by DrW3RK**
Include detailed explanation of how teleports work and the fees involved

KSM-teleport

### CODE REVIEW

File: docs/learn/learn-teleport.md [hunk 1]
```diff
@@ -14,25 +14,34 @@ and [SPREE modules][], which together ensure that assets are not lost or duplica
 
 ## How to Teleport
 
+Here are a couple of things that you need to understand before submitting a teleport transaction
```
Comment (gilescope) [inline docs/learn/learn-teleport.md:17]:
```suggestion
Here are a couple of things that you need to understand before submitting a teleport transaction:
```

File: docs/learn/learn-teleport.md [hunk 2]
```diff
@@ -14,25 +14,34 @@ and [SPREE modules][], which together ensure that assets are not lost or duplica
...
+
+- There will be a transaction fee imposed on the source chain. This fee is **not** automatically deducted from the teleport amount you plan to send. Your account balance should be greater than the teleport amount and the transaction fees. If you do not account for the sending fee, the teleport transaction will fail.
```
Comment (gilescope) [inline docs/learn/learn-teleport.md]: will fail (but you will not have been charged). <-- I presume?

File: docs/learn/learn-teleport.md [hunk 3]
```diff
@@ -14,25 +14,34 @@ and [SPREE modules][], which together ensure that assets are not lost or duplica
...
+
+- There will be a transaction fee imposed at the destination chain. This fee is automatically deducted from the teleport amount you send. After this fee is deducted, the remaining account balance on the destination chain should be greater than its Existential Deposit. If not, the teleport transaction will fail.
```
Comment (gilescope) [inline docs/learn/learn-teleport.md]: will fail (but you will have been charged in both locations?)

File: docs/learn/learn-teleport.md (hunk 2 above)
Comment (profradha) [reply docs/learn/learn-teleport.md]: Yeah, that is correct! Will add that detail

File: docs/learn/learn-teleport.md [hunk 4]
```diff
@@ -9,30 +9,51 @@ slug: ../learn-teleport
...
+
+- There will be a transaction fee imposed on the source chain. This fee is **not** automatically
+  deducted from the teleport amount you plan to send. Your account balance should be greater than
+  the teleport amount and the transaction fees. If you do not account for the sending fee, the
+  teleport transaction will fail.
+
+- There will be a transaction fee imposed at the destination chain. This fee is automatically
+  deducted from the teleport amount you send. After this fee is deducted, the remaining account
+  balance on the destination chain should be greater than its Existential Deposit. If not, the
+  teleport transaction will fail.
```
Comment (gilescope) [inline docs/learn/learn-teleport.md]: can you say in both cases if the failed transaction will have tx fees charged. I am guessing there can be no charge in the first case but you will be charged both tx fees on the remote failure?

File: docs/learn/learn-teleport.md (hunk 3 above)
Comment (profradha) [reply docs/learn/learn-teleport.md]: I could test again and confirm. As far as I can recollect, It did not charge me the receiving fee

**Review by gilescope — APPROVED** [review]
LGTM ( I have done said teleport )

File: docs/learn/learn-teleport.md (hunk 4 above)
Comment (profradha) [reply docs/learn/learn-teleport.md]: Will test and get back to you. I will tag you once the PR is ready for review

**Comment by DrW3RK** [comment]
Let me quickly verify the transaction fee situation and update the content to reflect it

**Comment by DrW3RK** [comment]
@gilescope You are right! If the transaction fails at the receiving end, you would lose both the sending fee and the teleported  amount. The doc is now updated with those details. Merging this PR

**Comment by gilescope** [comment]
Thanks for finding out - this rounds the docs off nicely.

**Pull request merged**